test/task_collection.test.js

```diff
@@ -0,0 +1,148 @@
+import TaskCollection from "../modules/task_collection";
+
+describe("Add to list", () => {
+  test("Test case 1", () => {
+    document.body.innerHTML =
+      "<div>" + '  <ul class="todo-list"></ul>' + "</div>";
+
+    const Test = new TaskCollection(true);
+    Test.addTaskToList("TaskTest_1 of Example 1");
+    Test.addTaskToList("TaskTest_1 of Example-2");
+    expect(Test.onSaveList()).not.toBeUndefined();
+    const list = document.querySelectorAll(".todo-list li");
+    expect(list).toHaveLength(2);
+  });
```

Comment:
- Hello @usorfaitheloho nice job on adding the tests suit but let's make you test more variants, so instead of repeating it let's add another test case with a localStorage to test the stored inputs.